packages/repository/src/common-types.ts

```diff
@@ -82,3 +82,12 @@ export type NamedParameters = AnyObject;
  */
 // tslint:disable-next-line:no-any
 export type PositionalParameters = any[];
+
+export interface Count {
```

Comment:
I know this is a straightforward interface, but can we add TSDocs to it to make it clear what it is?

Reply:
@b-admike Good call. Added comments.